src/StorageSync/StorageSync/RegisteredServer/RegisterServerCommand.cs

```diff
@@ -228,7 +228,7 @@ private RegisteredServer CreateRegisteredResourceInCloud(string resourceGroupNam
                 ClusterName = serverRegistrationData.ClusterName,
                 AgentVersion = serverRegistrationData.AgentVersion,
                 //ApplicationId = serverRegistrationData.ApplicationId.HasValue ? serverRegistrationData.ApplicationId.Value.ToString() : null,
-                ApplicationId = null,
+                ApplicationId = Guid.Empty.ToString(),
```

Comment:
This looks fine.

Separate, and optional: on line 173 in SyncServerRegistrationClientBase.cs in the Register function, there is this line, which can technically be commented out for now, since it is not used for Certificate registration -> v19

```c#
 Guid? applicationId = GetApplicationIdOrNull();
```

Reply:
I think that is fine. We are anyways retrieving mi capability.